web-ui/src/main/resources/templates/car-form.html

```diff
@@ -0,0 +1,14 @@
+<form>
+    <input name="carId">
+    <input name="carOwner">
+    <input name="carVin">
+    <input name="carPlateNumber">
+    <input name="carBrand">
+    <input name="carModel">
+    <input name="carGeneration">
+    <input name="carModification">
+    <input name="carEngine">
+    <input name="carYear">
+    <button type="submit">save</button>
+    <button type="reset">delete</button>
```

Comment:
I suggest give names to buttons also. 
type="reset" would erase form, check it here : https://www.w3schools.com/tags/att_button_type.aspon delete we we gona need 2 submit buttons.

Reply:
Done
As for the button, I thought it should clear the field before your explanation yesterday)